### ISSUE TITLE
Native export shows "File not downloaded"

### ISSUE THREAD

**Issue by nesbox** [nesbox/TIC-80#799]
without prompting for a place to save executable
0.70.6 on Ubuntu

**Comment by BuoYancYdabl** [comment]
This bug is even on android.

**Comment by ChildishGiant** [comment]
This bug is present on 0.70.6 Pro (from itch)

**Comment by makeworld-the-better-one** [comment]
I am still experiencing this, with the latest build (PRO I think) from the stable branch. Anything that can be done to fix?

**Comment by nesbox** [comment]
@makeworld-the-better-one what is your system, win linux mac?

**Comment by makeworld-the-better-one** [comment]
@nesbox I'm on Linux.

**Comment by RobLoach** [comment]
Can confirm. Compiled from source, Ubuntu 19.10...

```
> load cart.tic
> export sfx
file downloaded :)
> export native
file not download :|
```

**Comment by rottencandy** [comment]
Strangely, the `export native` command works if I launch the `tic80` binary after `cd`ing to it's installation directory(`/usr/local/bin` in my case) but doesn't work anywhere else, even though it's in my `$PATH`.

**Comment by nesbox** [comment]
Does it reproduce on the official build or your own build?
It's important.
Thanks.

**Comment by thratkin** [comment]
I can confirm that native export on Linux only works from the installation directory. I'm using my own (latest) build.

**Comment by rottencandy** [comment]
> Does it reproduce on the official build or your own build?
> It's important.
> Thanks.

I wasn't able to try the official build since I don't run a Debian based distro.

**Comment by maniek207** [comment]
Me too can confirm that native export on Linux (Debian 10) only works from installation dir (/usr/bin/tic80). I'm using latest build from website.

**Comment by nesbox** [comment]
Will be fixed with #1247
